rocAL/rocAL_pybind/amd/rocal/plugin/generic.py

```diff
@@ -38,50 +38,63 @@ def __init__(self, pipeline, tensor_layout = types.NCHW, reverse_channels = Fals
         self.batch_size = pipeline._batch_size
         if self.loader._name is None:
             self.loader._name = self.loader._reader
-        self.labels_size = ((self.bs*self.loader._numOfClasses) if (self.loader._oneHotEncoding == True) else self.bs)
-        self.out = self.dimensions = self.dtype = None
-        self.len = b.getRemainingImages(self.loader._handle)//self.bs # iteration length
+        self.labels_size = ((self.batch_size*self.loader._numOfClasses) if (self.loader._oneHotEncoding == True) else self.batch_size)
```

Comment:
- Please check and change the variables _numOfClasses and _oneHotEncoding to snake case if applicablle
- Please add space before and after * operator
- No need to explicitly check equality with true? if (self.loader._oneHotEncoding) should be sufficient i feel. Please check and change